src/main/resources/templates/db_scripts/schema.json

```diff
@@ -21,6 +21,13 @@
       "fromModuleVersion": 1.0,
       "withMetadata": true,
       "populateJsonWithId": true
+    },
+    {
+      "tableName": "voucher_lines",
+      "generateId": true,
+      "fromModuleVersion": 1.0,
+      "withMetadata": true,
+      "populateJsonWithId": true
```

Comment:
I think it makes sense to start using foreign keys to make sure the data is consistent with prerequisites i.e. line can be created only if voucher exists. The FK in this case would look like following:
```json
      "foreignKeys": [
        {
          "fieldName": "voucherId",
          "targetTable": "vouchers"
        }
      ]
```

But at the moment this cannot be done because voucher line does not have any knowledge which voucher it belongs to:
- [voucher_line.json](https://github.com/folio-org/acq-models/blob/master/mod-invoice-storage/schemas/voucher_line.json) - has no info about voucher
- [voucher.json](https://github.com/folio-org/acq-models/blob/master/mod-invoice-storage/schemas/voucher.json#L68) - contains array of voucher lines

Probably there is going to be created separate story on how to deal with such structure...

Reply:
I see this is handled in [MODINVOICE-49](https://issues.folio.org/browse/MODINVOICE-49)

Reply:
yeah, so we probably need a follow-up story for the FK part.

Reply:
https://issues.folio.org/browse/MODINVOSTO-18